x-pack/plugins/reporting/server/export_types/common/get_conditional_headers.test.ts

```diff
@@ -36,11 +34,7 @@ describe('conditions', () => {
       baz: 'quix',
     };
 
-    const conditionalHeaders = await getConditionalHeaders({
-      job: {} as ScheduledTaskParams<any>,
-      filteredHeaders: permittedHeaders,
-      config: mockConfig,
-    });
+    const conditionalHeaders = getConditionalHeaders(mockConfig, permittedHeaders);
```

Comment:
I went through many of these functions in `export_types/common` to use separate params instead of an options object. Also, use a convention to place the `config` parameter first.